website/source/api/system/replication-dr.html.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: "api"
+page_title: "/sys/replication - HTTP API"
+sidebar_current: "docs-http-system-replication-dr"
+description: |-
+  The '/sys/replication/dr' endpoint focuses on managing general operations in Vault Enterprise replication sets
```

Comment:
"replication sets" might be too generic here, we've used it with performance mode in the past too